Hometask_2/Hometask_2.py

```diff
@@ -0,0 +1,42 @@
+import random
+import string
+
+
+# Create the list dictionaries from 2 to 10.
+def create_dicts():
+    output = []  # Dicts list
+    for _ in range(random.randint(2, 10)):  # Dictionary counter
+        dictionary = {}
+        for _ in range(random.randint(1, 26)):  # Elements counter
+            # Key(random letter), value(random number from 0 to 100)
+            dictionary[random.choice(string.ascii_lowercase)] = random.randint(0, 100)
+        output.append(dictionary)  # Append every dictionary to the list
+    return output
+
+
+output = {}
+
+list_of_dicts = create_dicts()
+
+c = 1
+for cntr in list_of_dicts:
```

Comment:
Next time use enumerate() instead of additional variable ("c").
And what is "c"?